Bounding_volumes/include/CGAL/Min_ellipse_2/Optimisation_ellipse_2.h

```diff
@@ -24,6 +24,7 @@
 #include <CGAL/Conic_2.h>
 #include <CGAL/Optimisation/assertions.h>
 #include <CGAL/Kernel/global_functions_2.h>
+#include <CGAL/Kernel/Conic_misc.h>
```

Comment:
The already included `<CGAL/Conic_2.h>` already includes `Conic_misc.h`. Why it is needed here?